lib/config.go

```diff
@@ -47,6 +50,15 @@ type Config struct {
 	baseStore remoteStore
 }
 
+// yamlConfig specifies the optional settings taken from `.s3deploy.yml`
+type yamlConfig struct {
```

Comment:
The code in the PR looks correct, but since we already have a `fileConfig` struct and code that reads that from a YAML file, we should try to reuse that code. So I suggest that you just add these new fields to the same struct. And I suspect there should be some related tests you can expand on.

Reply:
Thanks for the review, I'm working on implementing your suggestions. :)